std_msgs/Header header
DetectedObject3D_kinect_circle[] detections